axlearn/experiments/text/gpt/c4_trainer.py

```diff
@@ -19,6 +19,22 @@
         --trainer_dir=$OUTPUT_DIR \
         --data_dir=$GS_ROOT/tensorflow_datasets \
         --mesh_selector=$INSTANCE_TYPE
+
+wget https://repo.anaconda.com/miniconda/Miniconda3-latest-Linux-x86_64.sh; \
+bash Miniconda3-latest-Linux-x86_64.sh; \
+bash
+conda create -n axlearn python=3.10; \
+conda activate axlearn; \
+git clone https://github.com/apple/axlearn.git; \
+cd axlearn; \
+git fetch origin pull/203/head:debug_gpu; \
```

Comment:
Is this intended for merge, or just to share w/ jax team?

Reply:
On GPU we may need to add some details about how we build jaxlib cuda (or see if we can repro with a pre-built jaxlib cuda version).

Reply:
Agreed. If you could follow up with a PR to fix the reproduction sequence, that will be great.